vue-frontend/src/components/portfolio/BrandingSection.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <section class="tw-bg-white tw-relative">
+    <div
+      class="container tw-py-24 sm:tw-pb-24 sm:tw-pt-44 md:tw-pb-36 2xl:tw-pt-56 tw-relative"
+    >
+      <div class="v2-header-3-text" v-html="response.title"></div>
+    </div>
+  </section>
+  <section class="background-image tw-relative">
+    <aspect-ratio height="56.26%">
+      <img :src="response.backgroundImage" />
+    </aspect-ratio>
+  </section>
+  <section class="tw-relative tw-bg-white">
```

Comment:
Is this section will have parallax effect or not?

Reply:
yes

Reply:
I am not able to see parallax effect